cmd/dt/auth.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import "github.com/spf13/cobra"
+
+var authCmd = &cobra.Command{
+	Use:           "auth",
```

Comment:
This is not very consistent with the existing group commands. Please have a look at `images` and `charts` for example for reference. Use should be something like "auth commands" or similar.

Reply:
This is exactly the same as is set for the `images` and `charts` commands. What do you mean?